app/styles/stepper.module.css

```diff
@@ -99,8 +105,10 @@
   .preview__label {
     font-size: 1.125rem;
   }
+}
 
+@media (max-width: 480px) {
   .user-input {
-    width: 20rem;
+    width: 95%;
```

Comment:
Nit: here you used `rem` https://github.com/Real-Dev-Squad/website-www/pull/648/files#diff-20059dd1a6ac67594a0a67d38caf2a0ebe8b580d32c7eb0c72545ba70c4d92eaR94
and here you user `percentage`

Is there any reason

Reply:
To give the size I want to use %

Reply:
give more context...not understand

Reply:
for mobile view, I don't want to give specific width as in mobile very pixel count to give 95% of it's parent width